cachito/workers/pkg_manager.py

```diff
@@ -12,37 +13,33 @@
 log = logging.getLogger(__name__)
 
 
-def resolve_gomod_deps(archive_path):
+def resolve_gomod_deps(archive_path, copy_cache=None):
```

Comment:
Optional: I prefer the keyword argument of `copy_cache_to` so it's clear a path is expected. In my opinion, `copy_cache` sounds like a bool is expected.